specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/examples/StorageAccountGetBlobInventoryPolicy.json

```diff
@@ -17,11 +17,11 @@
           "policy": {
```

Comment:
Just to confirm, the blob inventory policy don't have "Version" any more.
In the original GA design, it should have version as "1.0".

Reply:
Version is not a customer-facing property